test/modules/layers/test_codebook.py

```diff
@@ -4,250 +4,306 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-import unittest
 import warnings
 from collections import OrderedDict
 
+import pytest
+
 import torch
 from test.test_utils import assert_expected, set_rng_seed
 from torch import nn, tensor
 from torchmultimodal.modules.layers.codebook import Codebook
 
 
-class TestCodebook(unittest.TestCase):
```

Comment:
can we still contain all the tests in the `TestCodebook` class?

Reply:
Grouping tests by class becomes a must when the same test script contains multiple modules to test. When there's only one module, I think either way is fine technically.